composer.json

```diff
@@ -27,6 +27,7 @@
         "sonata-project/comment-bundle": "^3.1",
         "sonata-project/core-bundle": "^3.13.0",
         "sonata-project/datagrid-bundle": "^2.3",
+        "sonata-project/doctrine-extensions": "<=1.3.1",
```

Comment:
```suggestion
        "sonata-project/doctrine-extensions": "^1.3.1",
```

Reply:
@core23 but we need 1.0 - 1.3.1

Reply:
anyway it is sonata-project/doctrine-extensions issue
Sonata\Doctrine\Test\EntityManagerMockFactory::create try use $test->createMock() but it is protected method.

Reply:
You could use the fresh 1.5.1 release of the doctrine-extensions 🚀

Reply:
But its make errors

Reply:
> But its make errors

That's right, but you can fix the failing test easily